contrib/subprojects/tinyendian.wrap

```diff
@@ -0,0 +1,5 @@
+[wrap-git]
+directory = tinyendian
+
+url=https://github.com/ximion/tinyendian
```

Comment:
If it's tiny it could be added to the project as a source `dyaml.internal.endian` or something. It was already a request i sent a while ago (#26)